codalab_service.py

```diff
@@ -696,6 +698,7 @@ def start_services(self):
                         'server/default_user_info/parallel_run_quota',
                         self.args.user_parallel_run_quota,
                     ),
+                    ('server/public_workers', self.args.public_workers),
```

Comment:
This shouldn't be necessary, right?  We just need to pass `CODALAB_PUBLIC_WORKERS` through, which defining a `CodalabArg` already does.

Reply:
Yes. That's true. I removed this line.